Sluggo/View Controllers/AdminUserRolesTableViewController.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  AdminUserRolesTableViewController.swift
+//  Sluggo
+//
+//  Created by Stephan Martin on 5/17/21.
+//
+
+import UIKit
+
+class AdminUserRolesTableViewController: UITableViewController {
+
+    var identity: AppIdentity!
+    var member: MemberRecord?
+    var initialRole: String?
+    var role: String?
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureBarItems()
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        preSelect()
+    }
+
+    func configureBarItems() {
+        let doneAction = UIAction { _ in
+            self.changeMemberRecordRole()
+            self.navigationController?.popViewController(animated: true)
+        }
+        navigationItem.title = "User " + member!.owner.username
+        navigationItem.rightBarButtonItem = UIBarButtonItem(systemItem: .done, primaryAction: doneAction)
+    }
+
+    func preSelect() {
+        if member?.role == "AD" {
+            let indexPath = IndexPath(row: 1, section: 0)
+            tableView.selectRow(at: indexPath, animated: true, scrollPosition: .bottom)
+            role = "AD"
```

Comment:
Might be a good idea to move the roles formally into an enum with string raw values